### PR TITLE
Weekly Release 27.04.2016: Enterprise 3.1

### PR DESCRIPTION
**Pull request by bfallonf**
Weekly cherry-pick

### CODE REVIEW

File: architecture/infrastructure_components/web_console.adoc [hunk 1]
```diff
@@ -49,6 +51,20 @@ link:../../install_config/master_node_configuration.html#master-configuration-fi
...
+You can download and unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console] for use on Linux, MacOSX, and Windows clients
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
+endif::[]
+:
+
+image::about_page.png["About Page"]
+
```
Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: @bfallonf @vikram-redhat @ahardin-rh This should not get published into the OSE 3.1 docs yet. It's a 3.2 feature. I'm guessing this got pulled in by another commit or introduced maybe while resolving a merge conflict.

Comment (adellape) [inline architecture/infrastructure_components/web_console.adoc]: You can probably just yank it out here. https://github.com/openshift/openshift-docs/pull/1608 is the PR for this, and it will get picked for 3.2 at the appropriate time per its milestone.

File: install_config/revhistory_install_config.adoc [hunk 2]
```diff
@@ -4,6 +4,31 @@
...
 :experimental:
+== Wed Apr 27 2016
+
+// tag::install_config_wed_apr_27_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+n|link:../install_config/persistent_storage/persistent_storage_nfs.html[Configuring Persistent Storage -> Persistent Storage Using NFS]
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: Stray `n` at the front of this line? It doesn't appear to do anything in the rendered version, but... weird.

File: install_config/revhistory_install_config.adoc [hunk 3]
```diff
@@ -4,6 +4,31 @@
...
+|Updated the link:../install_config/persistent_storage/persistent_storage_nfs.html#nfs-export-settings[Export Settings] section to note the `no_wdelay` NFS export option.
+
+|link:../install_config/install/docker_registry.html[Installing -> Deploying a Docker Registry]
+|Updated the link:../install_config/install/docker_registry.html#registry-known-issues[Known Issues] section to note the `no_wdelay` NFS export option.
+
+|link:../install_config/http_proxies.html[Working with HTTP Proxies]
+|Added specific *_/etc/sysconfig_* files to the link:../admin_guide/managing_projects.html#configuring-hosts-for-proxies[Configuring Hosts for Proxies] section.
+
+|link:../install_config/http_proxies.html[Working with HTTP Proxies]
+|Added information explaining that OpenShift does not accept an asterisk as a wildcard attached to a domain suffix.
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: These two "Working with HTTP Proxies" notes could be combined as seen here:

https://github.com/bfallonf/openshift-docs/blame/enterprise-3.1/install_config/revhistory_install_config.adoc#L40-L52

File: welcome/revhistory_full.adoc [hunk 4]
```diff
@@ -4,6 +4,15 @@
...
 :experimental:
+== Wed Apr 27 2016
+.Admin Guide
+include::admin_guide/revhistory_admin_guide.adoc[tag=admin_guide_wed_apr_27_2016]
+
+.Architecture
+include::architecture/revhistory_architecture.adoc[tag=architecture_wed_apr_27_2016]
+
+.Install Config
+include::install_config/revhistory_install_config.adoc[tag=install_config_wed_apr_27_2016]
```
Comment (adellape) [inline welcome/revhistory_full.adoc]: @bfallonf Can you rename these "Admin Guide" and "Install Config" table titles? This is a current limitation of the release script. See how the full names are spelled out here: https://docs.openshift.com/enterprise/latest/welcome/revhistory_full.html

Also the script dumps this stuff above the intro sentence for this topic, so you have to cut/paste it back in place to fix it. See the current line 17 here below.

**Comment by adellape** [comment]
@bfallonf @vikram-redhat Added some comments inline for stuff to fix before merging/publishing.

**Comment by bfallonf** [comment]
@adellape Don't suppose you could double-check this and make sure it's all right? There seemed to be build issues coming from the registry_quickstarts docs and some of the files not being there. Other than that, the other things you brought up have been done. Thanks.

File: admin_guide/revhistory_admin_guide.adoc [hunk 5]
```diff
@@ -4,6 +4,22 @@
...
+== Wed Apr 27 2016
+
+// tag::admin_guide_wed_apr_27_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+|link:../admin_guide/manage_users.html[Managing Users]
+|New topic on how to add users, delete users, and view user and identity lists. (https://bugzilla.redhat.com/show_bug.cgi?id=1326595 [*BZ#1326595*])
```
Comment (adellape) [inline admin_guide/revhistory_admin_guide.adoc]: The space here breaks the link markup.

File: install_config/revhistory_install_config.adoc [hunk 6]
```diff
@@ -1,22 +1,46 @@
...
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+|link:../install_config/persistent_storage/persistent_storage_nfs.html[Configuring Persistent Storage -> Persistent Storage Using NFS]
+|Updated the link:../install_config/persistent_storage/persistent_storage_nfs.html#nfs-export-settings[Export Settings] section to note the `no_wdelay` NFS export option.
+
+|link:../install_config/install/docker_registry.html[Installing -> Deploying a Docker Registry]
+|Updated the link:../install_config/install/docker_registry.html#registry-known-issues[Known Issues] section to note the `no_wdelay` NFS export option.
+
+.2+|link:../install_config/http_proxies.html[Working with HTTP Proxies]
+|Added specific *_/etc/sysconfig_* files to the link:../admin_guide/managing_projects.html#configuring-hosts-for-proxies[Configuring Hosts for Proxies] section.
```
Comment (adellape) [inline install_config/revhistory_install_config.adoc]: This is linking to `../admin_guide/managing_projects.html`

Comment (ahardin-rh) [inline install_config/revhistory_install_config.adoc]: :scream_cat:That was a strange typo on my end. Sorry!

**Comment by bfallonf** [comment]
@adellape K. Done as suggested. Let me know if there's anything else.

**Comment by adellape** [comment]
:+1: Looks good, thanks.

**Comment by adellape** [comment]
@bfallonf Did a manual build, fixed some issues that was tripping up the Arch guide building (https://github.com/openshift/openshift-docs/pull/1968 https://github.com/openshift/openshift-docs/pull/1969 https://github.com/openshift/openshift-docs/pull/1971), and confirmed the changes are on the Customer Portal. 

The only remaining weirdness is the Instlal & Config guide's revhistory. The actual topic changes are fine, but the revhistory looks broken for the new date (Wed APr 27):

https://access.redhat.com/documentation/en/openshift-enterprise/version-3.1/installation-and-configuration/#revision-history

Comes out looking like:

```
Red Hat OpenShift Documentation Team 3.1 :experimental: == Wed Apr 27 2016
```

Hoping @tnguyen-rh and I can investigate more on Monday to fix that.

**Comment by tnguyen-rh** [comment]
@adellape
IIUC the asciidoc format specifies a blank line between the "head" of the source (which includes the attributes like `:experimental:`) and the "body" of the source.  To be excruciatingly correct, it *should be* an error to start the body immediately after the last attribute (as we've been doing for some cases to date), but the strictness of various processors is also variable (just to explain why this does not always show up).

Anyway, as part of adding the `do-release` [pragma](https://github.com/openshift/openshift-docs/commit/8b4eed39646c33d2cfc8de1e7a187ea07025c799), i've added a blank line after the last doc attribute where necessary.

**Comment by bfallonf** [comment]
@adellape @tnguyen-rh This looks fine in the published docs now, so I presume I don't have to followup with anything?

**Comment by adellape** [comment]
@bfallonf Yep, all good here.

**Comment by adellape** [comment]
^ @tnguyen-rh and I talked about that on Monday and he fixed it up.
